observability/dashboards/all/grafana.yaml

```diff
@@ -428,7 +322,7 @@ spec:
                 "uid": "logs-multena"
               },
               "editorMode": "code",
-              "expr": "{kubernetes_namespace_name=\"$namespace\"} | json | line_format \"Level: {{.level}}\\t\\tPod: {{.kubernetes_pod_name}}\\t\\tContainer: {{.kubernetes_container_name}}\\t\\tHost: {{.kubernetes_host}}\\t\\tPodIP: {{.kubernetes_pod_ip}}\\nMessage: {{.message}}\"",
+              "expr": "{kubernetes_namespace_name=`$namespace`} | json | line_format `{{ .message }}`  | line_format `{{ ._timestamp }} | {{ upper .level }} | {{ .kubernetes_pod_name }}: {{ .message }}`",
```

Comment:
kubernetes_pod_name bitte formatieren, zb. printf "%-20s"

Reply:
macht original 0 unterschied in der darstellung, außer dass die query schlechter zu lesen ist.